components/shared/infoBox/InfoBoxDetails.tsx

```diff
@@ -1,19 +1,25 @@
 import { motion } from "framer-motion"
 import React from "react"
 
+import { translateMaterialTypesStringForRender } from "@/components/pages/workPageLayout/helper"
 import InfoBoxItem from "@/components/shared/infoBox/InfoBoxItem"
-import { WorkFullWorkPageFragment } from "@/lib/graphql/generated/fbi/graphql"
+import {
+  GeneralMaterialTypeCodeEnum,
+  ManifestationWorkPageFragment,
+} from "@/lib/graphql/generated/fbi/graphql"
 import { getIsbnsFromManifestation } from "@/lib/helpers/ids"
-import { useSelectedManifestationStore } from "@/store/selectedManifestation.store"
-
-import { getTranslatedMaterialTypes } from "./helper"
 
 type InfoBoxDetailsProps = {
-  work: WorkFullWorkPageFragment
+  selectedManifestation: ManifestationWorkPageFragment
 }
 
-const InfoBoxDetails = ({ work }: InfoBoxDetailsProps) => {
-  const { selectedManifestation } = useSelectedManifestationStore()
+const InfoBoxDetails = ({ selectedManifestation }: InfoBoxDetailsProps) => {
+  // get selectedManifestation materialTypes and translate them for render
+  const materialTypeDisplays = selectedManifestation?.materialTypes.map(materialType => {
```

Comment:
Please remove `?` from the `selectedManifestation?.*` entries since we know that `selectedManifestation` is always defined.